### PR TITLE
Sec Webbing Rename And Map Overlay Removal

### PR DESCRIPTION
**Pull request by DieselMohawk**
## About the PR
 Renamed Security Webbing to Security Rig. Removed the Map Overlay for the Security Rig. Also gave myself the proper copyright since I messed that up next time.

## Why / Balance
 The map overlay made the baton float 4 feet away from the rig and the rig wasn't made for that overlay in mind.

## Technical details
 I made the Security Rig not be a child of the Security Belt so I could remove the Map Overlay component.

## Media

image
This is what it used to look like and what I'm removing.
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- tweak: Changed Security Webbing to Security Rig
- remove: Removed Security Webbing Map Overlay

### CODE REVIEW

**Comment by qrwas** [comment]
why are you doing this?!

**Comment by Saveliy360** [comment]
Not my words, but this is looks like you doing shitty resprite (let's be fr, new sec-webbing sprites SUCKS), realize that they do look shitty even ingame, and instead of removing it, you want to completely fucked up sprite mechanics for this item. Bro, stop. Just accept the thing - this resprite is not cool.

**Comment by ShadowCommander** [comment]
If you have a problem with changes, please give constructive criticism. What specific parts do you not like and *why*?

"Looks bad" or "This sucks" is not helpful and can't really be acted on by the person making the PR.

**Comment by Saveliy360** [comment]
> If you have a problem with changes, please give constructive criticism. What specific parts do you not like and _why_?
> 
> "Looks bad" or "This sucks" is not helpful and can't really be acted on by the person making the PR.

Okay, i give a constructivism why this is bad:

1. It was said before me, but new webbing looks more like striptease thing, not military gear.
2. Isn't that thing supposed to protect the wearer? Like, the point of the old webbing that is different from secbelt is that it give some protection with more capacity for battle stuff. And that's why it looked like a bulletproof vest in old sprites. But in the new one, the chest with all vital organs is open.
3. Even ingame that sprite isn't fit, so you need to delete the whole sprite mechanics for item. Why? Maybe I wouldn't complain on this so much if instead of deleting sprite mechanics, DieselMohawk do the actual RESPRITE of the Overlay. But just deleting the whole thing when your sprite is didn't work? Bruh.
4. The guy says that it's not "the child of secbelt anymore", but what then? It shares same mechanics, just with resprite. Maybe instead of just resprite, you can do some mechanics, that will make webbing/rig more different than just other secbelt AND respite? In one PR I mean. Like if I'm stupid with protection thing that I'm said in "2", add the ACTUAL protection to the webbing, that will be have actual sense, than just saying that it's not different from secbelt, and do nothing about it except the resprite.
5. I will repeat myself in "1", it's not fit the military style. The belts looks more like ropes, the whole rig is looking to "skinny" IMO. Like, in the old sprites it's look massive, and if you wear that it can actually feel like dangerous thing. I can imagine wearing this by security or military. But the new one looks more like just some suspenders, and not the battle gear.

File: Resources/Prototypes/Entities/Clothing/Belt/belts.yml [hunk 1]
```diff
@@ -608,16 +608,40 @@
...
-
+  - type: Storage
+    whitelist:
+      tags:
+        - CigPack
+        - Taser
+        - SecBeltEquip
+        - Radio
+        - Sidearm
+        - MagazinePistol
```
Comment (Dutch-VanDerLinde) [inline Resources/Prototypes/Entities/Clothing/Belt/belts.yml:628]: make an entity which has this storage component, then make the security rig and security belt inherit from it, this'll fix the bug and reduce copypaste

**Review by Dutch-VanDerLinde — CHANGES_REQUESTED** [review]

**Comment by DieselMohawk** [comment]
Alrighty, I got some good feedback and will be redesigning the Webbing's icon sprite. Expect to see that soon. I'll also look into the map overlay as well, assuming I can work with that at all. Thanks for the feedback, please try not to be as visceral next time.

File: Resources/Prototypes/Entities/Clothing/Belt/belts.yml (hunk 1 above)
Comment (DieselMohawk) [reply Resources/Prototypes/Entities/Clothing/Belt/belts.yml:628]: I'll look into it, thanks for the idea

**Pull request closed without merging**